docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+---
+layout: vNext
+title:  "Setting up and Running a Performance Benchmark on an existing network"
+categories: 
+permalink: /vNext/fabric-tutorial/tutorials-fabric-existing/
+order: 
+---
+## Table of Contents
+{:.no_toc}
+
+- TOC
+{:toc}
+
+## Overview
+This tutorial takes you through performance testing a smart contract on a pre-existing Fabric network using Caliper. 
+
+This tutorial follows from the test-network [fabric tutorial](https://hlf.readthedocs.io/en/latest/test_network.html). You need to have installed and instantiated the `asset-transfer-basic` smart contract (part two of the Fabric tutorial).
+
```

Comment:
Might be worth mentioning that a version of node is required for this tutorial, plus is you don't want to install Go then you can install the node version of asset-transfer-basic using the command `./network.sh deployCC -ccn basic -ccl javascript`.

Reply:
I'd also suggest using the javascript chaincode, since packaging it doesn't require 3rd party tools (and the ccenv image has every other dependency). While `go mod` requires a newer go version.

Reply:
added, along with a code block fast start snippet